<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareDialog</class>
 <widget class="QDialog" name="CompareDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1764</width>
    <height>1043</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Compare Cases</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1744</width>
        <height>1023</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1740</width>
        <height>1014</height>
       </size>
      </property>
      <widget class="QGroupBox" name="viewGroupBox">
       <property name="geometry">
        <rect>
         <x>946</x>
         <y>-1</y>
         <width>182</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>View</string>
       </property>
       <widget class="QLabel" name="viewBladderCheckboxShadowLabel">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>37</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Bladder</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>12</y>
          <width>161</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="viewRectumCheckBox">
           <property name="text">
            <string>Rectum</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="viewPTVCheckBox">
           <property name="text">
            <string>PTV</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="viewBladderCheckBox">
           <property name="text">
            <string>Bladder</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="viewFemoralHeadsCheckBox">
           <property name="text">
            <string>Femoral Heads</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="projectionDisplayGroupBox">
       <property name="geometry">
        <rect>
         <x>565</x>
         <y>-1</y>
         <width>375</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Beam's Eye View Projection</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="transparencyLabel">
          <property name="text">
           <string>%Transparency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="transparencySpinBox">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="transparencySlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="maxTransparencyLabel">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gantryAnglePushButton">
          <property name="text">
           <string>Gantry Angle</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="ctAxisGroupBox">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>-1</y>
         <width>465</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>CT</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="axialRadioButton">
          <property name="text">
           <string>Axial</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sagittalRadioButton">
          <property name="text">
           <string>Sagittal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="coronalRadioButton">
          <property name="text">
           <string>Coronal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sliceSelectionLabel">
          <property name="text">
           <string>Z slice:</string>
          </property>
          <property name="buddy">
           <cstring>sliceSelectionSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliceSelectionSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="isocenterPushButton">
          <property name="text">
           <string>Isocenter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="autoPlayPushButton">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Auto-play</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="DVHGroupBox">
       <property name="geometry">
        <rect>
         <x>1214</x>
         <y>-1</y>
         <width>295</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Dose Volume Histogram</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="overlayDVHCheckBox">
          <property name="text">
           <string> Overlay On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="overlaySelectionPushButton">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Select Overlay</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeSelectedOverlayPushButton">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Remove Selected</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>56</y>
         <width>1686</width>
         <height>932</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="queryGroupBox">
            <property name="maximumSize">
             <size>
              <width>81</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Query</string>
            </property>
            <widget class="QPushButton" name="queryInstitutionPushButton">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>106</y>
               <width>59</width>
               <height>23</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>59</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>59</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Duke</string>
             </property>
            </widget>
            <widget class="QLabel" name="queryCaseInstitutionLabel">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>206</y>
               <width>59</width>
               <height>24</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>59</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>59</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Duke</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="queryCaseNumberLabel">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>226</y>
               <width>59</width>
               <height>24</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>59</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>59</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>#666</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QSpinBox" name="querySelectSpinBox">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>408</y>
               <width>39</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>12</number>
             </property>
            </widget>
            <widget class="QSpinBox" name="sliceSelectionSpinBox">
             <property name="geometry">
              <rect>
               <x>22</x>
               <y>40</y>
               <width>39</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
            </widget>
            <widget class="QLabel" name="maxSliceLabel">
             <property name="geometry">
              <rect>
               <x>31</x>
               <y>43</y>
               <width>20</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>max</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="originCheckBox">
             <property name="geometry">
              <rect>
               <x>21</x>
               <y>44</y>
               <width>51</width>
               <height>17</height>
              </rect>
             </property>
             <property name="text">
              <string>Axes</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="flatShadedCheckBox">
             <property name="geometry">
              <rect>
               <x>8</x>
               <y>48</y>
               <width>79</width>
               <height>17</height>
              </rect>
             </property>
             <property name="text">
              <string>Flat shaded</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QVTKWidget" name="queryCTImageWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>460</width>
              <height>460</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <widget class="QLabel" name="dvhLabel">
             <property name="geometry">
              <rect>
               <x>-78</x>
               <y>40</y>
               <width>25</width>
               <height>25</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Dose Volume Histogram</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QVTKWidget" name="queryProjectionWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>460</width>
              <height>460</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QVTKWidget" name="overlayDVHWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>663</width>
              <height>460</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="matchGroupBox">
            <property name="minimumSize">
             <size>
              <width>79</width>
              <height>460</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Match</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QPushButton" name="matchInstitutionPushButton">
                 <property name="minimumSize">
                  <size>
                   <width>59</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>59</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Duke</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="matchSelectSpinBox">
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>500</number>
                 </property>
                 <property name="value">
                  <number>12</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QVTKWidget" name="matchCTImageWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>460</width>
              <height>460</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QVTKWidget" name="matchProjectionWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>460</width>
              <height>460</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QVTKWidget" name="matchDVHWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>663</width>
              <height>460</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>990</y>
         <width>1687</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="historyPushButton">
          <property name="text">
           <string>History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeCurrentMatchPushButton">
          <property name="text">
           <string>Remove Current</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="savePushButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1259</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
